test/shopify_app/scripttags_manager_test.rb

```diff
@@ -5,7 +5,8 @@ class ShopifyApp::ScripttagsManagerTest < ActiveSupport::TestCase
   setup do
     @scripttags = [
       {event: 'onload', src: 'https://example-app.com/fancy.js'},
-      {event: 'onload', src: 'https://example-app.com/foobar.js'}
+      {event: 'onload', src: 'https://example-app.com/foobar.js'},
+      {event: 'onload', src: ->() { 'https://example-app.com/dynamic.js' } }
```

Comment:
What do you think about expanding this test case to be a bit more dynamic? I think it might help reveal the purpose of making these callable